### ISSUE TITLE
Update CDI sections for Jakarta EE 10

### ISSUE THREAD

**Issue by kito99** [jakartaee/jakartaee-tutorial#55]

**Comment by kito99** [comment]
@antoinesd and @starksm64 please take a look and let me know (a) if this  sounds correct, and (b) if you or someone on the committer team would like to contribute. Thanks!

**Comment by jhammen** [comment]
for sure, will take a look

**Comment by jhammen** [comment]
@kito99 feel free to comment on the commit above, changes reflected here: https://jhammen.github.io/jeetutorial/current/cdi/cdi-basic/cdi-basic.html#_cdi_lite_vs_cdi_full

**Comment by kito99** [comment]
@jhammen more background info here: https://newsroom.eclipse.org/eclipse-newsletter/2022/october/jakarta-ee-10-provides-first-jakarta-feature-updates

**Comment by kito99** [comment]
@jhammen the change above includes just the CDI Lite verbiage, correct?

**Comment by jhammen** [comment]
yes a commit per bullet point so far, second two points still TBD

**Comment by kito99** [comment]
@starksm64 looks like I can't assign this to you, but feel free to work with @jhammen on this; he has kindly offered to help but is not a CDI subject matter expert.

**Comment by jhammen** [comment]
Honestly haven't gotten too far on this one, just the Lite vs Full intro section and updated a few links - I just made a PR in case you all want to use that

Only other thing I did was start taking notes on the sections themselves, the previous Basic vs Advanced section split doesn't line up cleanly with the Lite vs Full split. So it looks like some serious rewriting or at least moving things around.

I'm not really comfortable with that so I'm going to officially step back here and defer to whatever you two decide. I'm available as a resource if there are more clear tasks to do (but less so during the holiday season).

**Comment by kito99** [comment]
ok, thanks so much, @jhammen! 

@starksm64 will you be able to start working on this before the holidays or early in the new year?

**Comment by kito99** [comment]
@starksm64 what is the status of this?

**Comment by kito99** [comment]
@starksm64 are you still willing to take this on?
